src/domains/wallet/components/VerifyMessage/VerifyMessage.tsx

```diff
@@ -77,7 +78,8 @@ export const VerifyMessage = ({ profileId, walletId, onSubmit, onCancel, signato
 			<div data-testid="noverify-address__content">
 				<FormField name="message" className="mt-8">
 					<FormLabel label={t("COMMON.MESSAGE")} />
-					<Input
+					<InputDefault
+						type="text"
```

Comment:
I thought this property was removed in a recent PR 🤔 , applies to multiple instances